tests/AbstractProcessTest.php

```diff
@@ -96,10 +103,16 @@ public function testProcessWithDefaultCwdAndEnv()
 
     public function testProcessWithCwd()
     {
-        $cmd = $this->getPhpBinary() . ' -r ' . escapeshellarg('echo getcwd(), PHP_EOL;');
+        $cmd = $this->getPhpCommandLine('echo getcwd(), PHP_EOL;');
+
+        if (defined('PHP_WINDOWS_VERSION_BUILD')) {
+            $testCwd = 'C:\\';
+        } else {
+        	$testCwd = '/';
+        }
```

Comment:
Yes, exactly. on Windows , `getcwd()` will return "C:\", even if you pass '/' as the default cwd for the command.